spec/features/upload_rap_sheet_spec.rb

```diff
@@ -118,7 +118,7 @@
           'Field13' => '19514114',
           'Field43' => 'Yes',
           # 1203 dismissal form for arson case
-          '3.Field13' => '44050',
+          '4.Field13' => '44050',
```

Comment:
There's a number of places where fields have overlapping names and we are targetting them with a pdfTK syntax. I think these were the locked PDFs that we couldn't edit to have human-friendly field names but ooooof if we ever add fields we need to edit these places